codecov_cli/helpers/git_services/github.py

```diff
@@ -0,0 +1,32 @@
+import json
+
+import requests
+
+
+class Github:
+    api_url = "https://api.github.com"
+    api_version = "2022-11-28"
+
+    def get_pull_request(self, slug, pr_number):
```

Comment:
i think this is tested indirectly in `test_git.py` but it'd be nice if it had its own tests

Reply:
I agree, added tests